src/solver/utils/mps_utils.cpp

```diff
@@ -16,14 +17,14 @@ using namespace Antares::Data;
 
 constexpr size_t OPT_APPEL_SOLVEUR_BUFFER_SIZE = 256;
 
-static void printHeader(FILE* Flot, int NombreDeVariables, int NombreDeContraintes)
+static void printHeader(Clob& Flot, int NombreDeVariables, int NombreDeContraintes)
```

Comment:
In this source file, we could replace **Clob** with **std::string**, we would get rid of Yuni and the code would be faster to execute (no attention to performance issues was given in **Yuni::Clob**)